footer-featured.php

```diff
@@ -24,6 +21,7 @@
 				</div>
 			</div>
 		</div>
+	</div>
```

Comment:
Is this extra closing div necessary?

Reply:
This closing div closes the container div opened in `header-features.php` and the closing div right above this closes the `<div id="feature-story">` opened in `featured-single-post.php`. Should I be closing the div.container in `featured-single-post.php` and opening a new one in this file above div#footer?